participants.md

```diff
@@ -5,3 +5,5 @@ Classes on JS, React, redux and other tools for Front end development.
 ### Participants
 
 Trainer: Radheshyam Etikala
+
+Tejaswini Jadhav
```

Comment:
Maybe you can add a '-' here so that it shows as a list

Reply:
Updated!